projects/ion/src/lib/sidebar/sidebar.component.ts

```diff
@@ -58,4 +58,14 @@ export class IonSidebarComponent {
     unselectAllItems(this.items);
     callItemAction(this.items, groupIndex);
   }
+
+  public handleLogoClick(): void {
+    if (this.logoAction) {
+      this.logoAction();
+    }
+
+    if (this.closeOnSelect) {
+      this.toggleSidebarVisibility();
+    }
```

Comment:
If the 2 actions can be run, LGTM.

Reply:
I've made a build and tested in caná project, it worked just fine redirecting and closing after :smile: 

[Gravação de tela de 2023-12-01 12-53-55.webm](https://github.com/Brisanet/ion/assets/138057627/352ea10b-4478-4a24-89b1-656161214b2d)

Reply:
LGTM.